src/osd/ReplicatedPG.h

```diff
@@ -1,4 +1,5 @@
 // -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*- 
+// vim: ts=8 sw=2 smarttab
```

Comment:
I guess this line was missing? But I'm not sure why you're adding it in a commit/pull that otherwise doesn't touch the file.